templates/components/password.build.hbs

```diff
@@ -9,7 +12,7 @@
   :required="required"
 />
```

Comment:
I'm blinding coding HTML here, but I think when a password is autogenerated and used, the build should include a message below the field.
`<p class="help-block">A password will be automatically generated</p>`